src/openfermion/transforms/_conversion.py

```diff
@@ -432,6 +432,99 @@ def _majorana_term_to_fermion_operator(term):
     return converted_term
 
 
+def get_majorana_operator(operator):
+    """
+    Convert to MajoranaOperator.
+
+    Args:
+        operator (PolynomialTensor,
+            DiagonalCoulombHamiltonian or
+            FermionOperator): Operator to write as Majorana Operator.
+
+    Returns:
+        majorana_operator: An instance of the MajoranaOperator class.
+
+    Raises:
+        TypeError: If operator is not of PolynomialTensor,
+            DiagonalCoulombHamiltonian or FermionOperator.
+    """
+    if not isinstance(
+            operator,
+        (PolynomialTensor, DiagonalCoulombHamiltonian, FermionOperator)):
+        raise TypeError('{} cannot be converted to MajoranaOperator'.format(
+            type(operator)))
+    try:
+
+        return _fermion_operator_to_majorana_operator(operator)
+
+    except:
+
+        return _fermion_operator_to_majorana_operator(
+            get_fermion_operator(operator))
+
+
+def _fermion_operator_to_majorana_operator(fermion_operator):
+    """
+    Convert FermionOperator to MajoranaOperator.
+
+    Auxiliar function of get_majorana_operator.
+
+    Args:
+        fermion_operator (FermionOperator): To convert to MajoranaOperator.
+
+    Returns:
+        majorana_operator object.
+
+    Raises:
+        TypeError: if input is not a FermionOperator.
+    """
+    if not isinstance(fermion_operator, FermionOperator):
+        raise TypeError('Input a FermionOperator.')
+
+    majorana_operator = MajoranaOperator()
+
+    for term, coeff in fermion_operator.terms.items():
+
+        converted_term = _fermion_term_to_majorana_operator(term)
+        converted_term *= coeff
+        majorana_operator += converted_term
+
+    return majorana_operator
+
+
+def _fermion_term_to_majorana_operator(term):
+    """
```

Comment:
Put the mathematical description here.  There are a number of choices one can make when ordering majorana modes and which ones correspond to which fermion.  I see you've made the even + odd = fermion choice.  Be explicit about this in the docs.  also add type hint and return type.

Reply:
I added a description of the convention used also in the function intented for users (get_majorana_operator(operator))